src/store/reducers/helpers.ts

```diff
@@ -0,0 +1,14 @@
+export const getBlockchainByDatasource = (
```

Comment:
Create a file `blockchainSelectors.ts` and paste this code in there. `helpers.ts` is too ambiguous.